### PR TITLE
Created branch for GPS recording

### LINKED ISSUE #65: Implement an Activity where the user can start, pause, and stop a recording of GPS coordinates

### LINKED ISSUE #66: Add a MapView to the GPS recording Activity to be able to visualize the path afterwards

### PR DESCRIPTION
**Pull request by bitshrine**
Still in progress :)

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -19,10 +19,10 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.movemeet">
+        <activity android:name=".map.GPSRecordingActivity"></activity>
         <activity
             android:name=".EditProfileActivity"
-            android:windowSoftInputMode="adjustPan" >
-        </activity>
+            android:windowSoftInputMode="adjustPan"></activity>
```
Comment (RoxaneBurri) [inline app/src/main/AndroidManifest.xml:25]: You only delete line in the file ?

File: app/src/androidTest/java/com/sdp/movemeet/map/GPSRecordingActivityTest.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    @Rule
+    public GrantPermissionRule CoarseLocationPermissionRule = GrantPermissionRule.grant(Manifest.permission.ACCESS_COARSE_LOCATION);
+
+
+    @Rule
+    public ActivityScenarioRule<GPSRecordingActivity> testRule = new ActivityScenarioRule<>(GPSRecordingActivity.class);
+
+    private UiDevice device;
+
+    private Task<Void> mockTask;
```
Comment (RoxaneBurri) [inline app/src/androidTest/java/com/sdp/movemeet/map/GPSRecordingActivityTest.java:69]: Why do you use mock ?

**Review by RoxaneBurri — APPROVED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (bitshrine) [reply app/src/main/AndroidManifest.xml:25]: I had so solve a merge conflict in the file and probably deleted the extra characters by accident

File: app/src/androidTest/java/com/sdp/movemeet/map/GPSRecordingActivityTest.java (hunk 2 above)
Comment (bitshrine) [reply app/src/androidTest/java/com/sdp/movemeet/map/GPSRecordingActivityTest.java:69]: I'm mocking the `FusedLocationProviderClient` and injecting my own code when the `.requestLocationUpdates()` method is called. I had to create a mock Task<Void> instance to return something consistent with the method signature, after having run my own testing code.

**Review by kotleta2007 — APPROVED** [review]

**Pull request merged**